Python/Product/Cookiecutter/Model/GitClient.cs

```diff
@@ -52,6 +52,9 @@ class GitClient : IGitClient {
             }
 
             var arguments = new string[] { "clone", repoUrl };
+            if (!ExistsOnPath(_gitExeFilePath)) {
+                throw new FileNotFoundException();
```

Comment:
As Pavel mentioned offline, it would be better to run `git verion` to make sure git can actually run. No need to check the path.

Also, I believe VS now comes with git built in, so using assuming git.exe is on the path no longer seems necessary, we can just use the one that comes with VS. But if you don't want to make that change, I'm fine with just trying to run git.